README.md

```diff
@@ -292,6 +292,7 @@ Contributions are what make the open source community such an amazing place to b
  2. From within your dashboard, go to the [developers](https://dashboard.daily.co/developers) tab.
  3. Copy your API key.
  4. Now paste the API key to your .env file into the `DAILY_API_KEY` field in your .env file.
+ 5. If you have a [Daily Scale Plan](https://www.daily.co/pricing)  can also enable the ability to record Daily video meetings. To do so, set the `DAILY_SCALE_PLAN` environment variable to `enable` 
```

Comment:
Could we tell people to set the variable to `true` for consistency? I don't think it should affect the rest of this code if people set it to true

Reply:
Sure, can. It's updated thanks!